rllib/algorithms/a2c/tests/test_a2c.py

```diff
@@ -26,7 +26,7 @@ def test_a2c_compilation(self):
 
         # Test against all frameworks.
         for _ in framework_iterator(config, with_eager_tracing=True):
-            for env in ["CartPole-v1", "Pendulum-v1", "PongDeterministic-v0"]:
+            for env in ["ALE/Pong-v5", "CartPole-v1", "Pendulum-v1"]:
```

Comment:
Note for everything below: All Atari env strings have been replaced by the new gymnasium notation using the "ALE/" prefix (+ options like frameskip and deterministic go into the env_config kwargs).

Reply:
kk